ui/components/app/detected-token/detected-token-values/detected-token-values.js

```diff
@@ -46,7 +46,7 @@ const DetectedTokenValues = ({
           {`${balanceString || '0'} ${token.symbol}`}
         </Typography>
         <Typography variant={TYPOGRAPHY.H7} color={COLORS.TEXT_ALTERNATIVE}>
-          {formattedFiatBalance || '$0'}
+          {formattedFiatBalance}
```

Comment:
should we still show `$0` if the user has the `useCurrencyExchange` set to true but `formattedFiatBalance` is undefined?

Reply:
I have made change to still show default `$0` if `useCurrencyRateCheck` is true. Ideally is `useCurrencyRateCheck` is true and we do not get rate we show show something like `N/A` but we can check that in a separate PR.

Reply:
It's not clear to me when `formattedFiatBalance` is falsy. Is there another case this may occur aside from not receiving a rate yet? If so, can we create a separate issue ticket and/or add a comment? If not, why not apply `N/A` instead of `$0` now? 

@jpuri @NiranjanaBinoy

Reply:
I think for the `formattedFiatBalance` to be false the conversion rate has to be missing, or there is an issue with balance fetching while viewing the page. I don't remember exactly, but I think we might have added the `$0` after checking with the design time at the time of token detection v2. Are we sure changing that to `N/A` will not create any other confusion for the user?

Reply:
Will it be okay if we do that in a follow-up PR? This PR needs to go into v10.24 to fix an existing issue.

Reply:
Warnings
⚠️	
Libraries/Components/Button.js#L14 - Libraries/Components/Button.js line 14 – Requires should be sorted alphabetically (lint/sort-imports)